### DISCUSSION TITLE
Path - runOnExit runOnPublishClose?

### DISCUSSION THREAD

**Discussion by gluedots** [bluenviron/mediamtx#554] [Questions] [answered]
## Describe the feature

Hello, would it be possible to have something like a runOnExit or runOnPublishClose config option?

I would like to do some cleanup when publishing stops or path is closed.

Thanks!

**Comment by aler9** [comment, marked as answer]
It's already possible to do this by creating a bash script like this:

```sh
#!/bin/bash
trap 'echo publisher disconnected' QUIT
echo publisher connected
sleep 86400 &
wait
```

and placing the script path into `runOnPublish`. The content inside `trap` will be called when the publisher disconnects, while the next line will be called when the publisher connects.

[a comment by an automated account is not shown]
